controllers/comments.controllers.js

```diff
@@ -1,10 +1,18 @@
+const { checkExists } = require("../db/seeds/utils");
 const { selectCommentsByArticleId } = require("../models/comments.models");
 
 exports.getCommentsByArticleId = (req, res, next) => {
-    const { article_id } = req.params;
-    selectCommentsByArticleId(article_id)
-      .then((comments) => {
-        res.status(200).send({ comments });
-      })
-      .catch(next);
-  };
\ No newline at end of file
+  const { article_id } = req.params;
+  const articlePromises = [selectCommentsByArticleId(article_id)];
+
+  if (article_id) {
```

Comment:
The article id here isn't optional, you could resolve all of this in a Promise.all()